targets/TARGET_Samsung/security_subsystem/drivers/sss_driver_sha2.h

```diff
@@ -0,0 +1,135 @@
+/**************************************************************************/
+/* Copyright (c) Samsung Electronics Co., Ltd. All rights reserved.       */
+/*                                                                        */
```

Comment:
This file as well needs more permissive license. Please review all files under security_subsystem

Reply:
@0xc0170. Thanks for pointing this out as well.

@jh6186
Please note that these are the files without Apache 2 declaration and the current copyright is restricted. Please consider moving these file's license to Apache 2 with SPDX mentioning.
```
> targets/TARGET_Samsung/security_subsystem/drivers/mb_cmd_hash.c
> targets/TARGET_Samsung/security_subsystem/drivers/mb_cmd_hash.h
> targets/TARGET_Samsung/security_subsystem/drivers/sss_common.h
> targets/TARGET_Samsung/security_subsystem/drivers/sss_driver_sha2.c
> targets/TARGET_Samsung/security_subsystem/drivers/sss_driver_sha2.h
> targets/TARGET_Samsung/security_subsystem/drivers/sss_map.h
> targets/TARGET_Samsung/TARGET_SIDK_S5JS100/spi_api.c
```

Reply:
Also, this is the same as the above review item where every Samsung target code is marked with SPDX Apache 2.0. https://github.com/ARMmbed/mbed-os/pull/12106/commits/1a9049f5d94dfda115501cfb01350106226ff043